sycl/include/sycl/ext/oneapi/experimental/user_defined_reductions.hpp

```diff
@@ -9,6 +9,7 @@
 #pragma once
 
 #include <sycl/detail/defines.hpp>
+#include <sycl/ext/oneapi/experimental/group_helpers_sorters.hpp>
```

Comment:
I'm most likely missing something, why we need to include this header for sorting algorithms into reductions header?

Reply:
`group_with_scratchpad` is defined in the sorting extension, UDR extension depends on that one for it.

Reply:
I see, thanks!